src/components/Modal/Modal.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { MdClose as CloseIcon } from 'react-icons/md'
+import Title from 'components/Title/Title'
+import Text from 'components/Text/Text'
+import Container from 'components/Container/Container'
+import styles from 'components/Modal/modal.module.scss'
+
+const Modal = ({ item, onCloseModal, ...props }) => {
+  const handleClick = () => onCloseModal()
+  const ESTADO = {
```

Comment:
Ojo que `ESTADO` quedó en español. Por otro lado, los enums pueden ir por fuera del componente, lo que es recomendable para que este quede más "limpio" y simple, así que extraerlo y ponerlo por fuera.

```suggestion
  const TODO_STATUS = {
```